include/tvm/runtime/packed_func.h

```diff
@@ -957,6 +964,25 @@ inline const char* TypeCode2Str(int type_code) {
   }
 }
 
+inline const char* DeviceType2Str(int device_type) {
```

Comment:
Actually something similar is defined here:
https://github.com/dmlc/tvm/blob/master/include/tvm/runtime/device_api.h#L205